capstone/src/main/webapp/script.js

```diff
@@ -55,6 +58,27 @@ function lookupPollingPlace(address) {
   return fetch(civicinfo).then(response => response.json());
 }
 
+//puts the address into the DOM
```

Comment:
Update this to have a better description, something like "Appends polling location addresses to the DOM."